src/sounds.py

```diff
@@ -0,0 +1,80 @@
+import parselmouth
+
+import argparse
+import pathlib
+import math
+import os
+
+class Sound:
+    def __init__(self, note, timestamp, duration):
+        self.note = note
+        self.timestamp = timestamp
+        self.duration = duration
+
+    @property
+    def symbol(self):
+        if self.note == None:
+            return 'None'
+        symbols = ['C', 'C#', 'D', 'D#', 'E', 'F', 'F#', 'G', 'G#', 'A', 'A#', 'H']
+        return symbols[self.note]  
+
+    def __str__(self):
+        return f"{round(self.timestamp, 3)}: {self.symbol}\n"
+
+    def __ne__(self, other):
+        return self.note != other.note
+
+def frequency_to_note(frequency):
+    '''
+    A4 440 Hz
+    '''
+    if frequency == 0:
+        return None
+    return round(12*math.log2(frequency/440.0)+45) % 12
+
+
+class Sounds:
+    def __init__(self, file):
+        self._sounds = []
```

Comment:
It should be public

Reply:
From ur previous comment ("python naming convention") I understood that it should be private. On my way to fix this

Reply:
Fixed